File: library/src/test/java/com/ebr163/inspector/rule/NotNullRuleTest.java
package com.ebr163.inspector.rule;

import org.junit.Test;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

/**
 * Created by Bakht
 * on 12.01.2018.
 */
public class NotNullRuleTest {

    @Test
    public void verify() throws Exception {
        NotNullRule<String> rule = new NotNullRule<>("error");
        assertFalse(rule.verify(null));
        assertTrue(rule.verify(""));
    }
}

File: library/src/test/java/com/ebr163/inspector/rule/NotNullRuleTest.kt
package com.ebr163.inspector.rule

import org.junit.Test

import org.junit.Assert.assertFalse
import org.junit.Assert.assertTrue

/**
 * Created by Bakht
 * on 12.01.2018.
 */
class NotNullRuleTest {

    @Test
    @Throws(Exception::class)
    fun verify() {
        val rule = NotNullRule<String>("error")
        assertFalse(rule.verify(null))
        assertTrue(rule.verify(""))
    }
}